wellcomeml/ml/transformers_tokenizer.py

```diff
@@ -99,14 +100,19 @@ def tokenize(self, text):
             raise NotImplementedError
 
     def encode(self, text):
-        if type(text) == list:
+        if type(text) in [list, np.ndarray]:
```

Comment:
I think people usually prefer `isinstance` for these comparisons because then it also holds for sub-classes. 

Also, maybe you only need it to be an iterable of any sort?

Reply:
I somehow do not use `isinstance` as often as I should. In this case though can `list` or `np.ndarray` be subleased in a meaningful way?

How would you check for an iterable? `hasattr(text, __iter__)`?

Reply:
Oh sorry, I missed this comment. Yeah, I don't think there are any sub-classes of list or Numpy arrays that I can recall of. It's a general practice, in other repositories that I used typed python and `mypy` checks, it fails to identify `type(x) = ` kind of comparisons.

I used this to check for iterables before (https://docs.python.org/3/library/typing.html#user-defined-generic-types):
```python
from collections.abc import Iterable
isinstance(x, Iterable)
```